addons/adminMenu/functions/fnc_isDebugConsoleAllowed.sqf

```diff
@@ -0,0 +1,16 @@
+private _enableDebugConsole = [
+    "DebugConsole",
+    getMissionConfigValue ["enableDebugConsole", 0]
+] call (missionNamespace getVariable "BIS_fnc_getParamValue");
+
+ // Always allowed by mission config or mission param
+if (_enableDebugConsole isEqualTo 2 || {hasInterface && isServer}) exitWith { true };
+
+// In single player editor
+if (!isMultiplayer && { allDisplays find findDisplay 26 in [0, 1] || allDisplays find findDisplay 313 in [0, 1] }) exitWith { true };
```

Comment:
If it's not isMultiplayer , we should have already matched with the line above: `hasInterface && isServer`.
So this isn't necessary but won't hurt.
I guess `DebugConsole = 0` will be ignored, but I see no problem with that